src/properties/property-effects.html

```diff
@@ -1254,16 +1532,35 @@
       /**
        * Implements `PropertyAccessors`'s properties changed callback.
        *
-       * Base implementation simply iterates the list of all property effects
-       * and runs them in the order they were added.
+       * Runs each class of effects for the batch of changed properties in
+       * a specific order (compute, propagate, reflect, observe, notify).
        *
        * @override
        */
       _propertiesChanged(currentProps, changedProps, oldProps) {
-        for (let p in changedProps) {
-          let effects = this.__propertyEffects[p];
-          runEffects(this, p, changedProps[p], oldProps[p], effects);
-        }
+        // ----------------------------
```

Comment:
Moved from `batched-effects` to `property-effects`.  Simplified to remove handling of notify re-entry support, and moves notify to end.